cypress/integration/plugins/reports-dashboards/01-create.spec.js

```diff
@@ -3,24 +3,24 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { visitReportingLandingPage } from "../../../utils/plugins/reports-dashboards/constants";
+import { visitReportingLandingPage, SAMPLE_DATA_WAIT_TIME } from "../../../utils/plugins/reports-dashboards/constants";
```

Comment:
FYI: you can export your plugin constants in the [central constants file](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/utils/constants.js) and then simplify all of your imports to just be `from "../../../utils/constants"`